modules/apps/headless/headless-admin-user/headless-admin-user-impl/rest-openapi.yaml

```diff
@@ -1068,6 +1068,11 @@ components:
                     format: int64
                     readOnly: true
                     type: integer
+                cms:
+                    description:
+                        Whether the shared asset will be shown on the cms
+                    readOnly: true
+                    type: boolean
```

Comment:
Why are you exposing this here? We only need it in the filter, so adding it to the entity model should be enough.

Reply:
Because I was wrong on my understanding.

Thanks for pointing at it :)